product_dimension/product.py

```diff
@@ -21,7 +21,7 @@
 
 
 class Product(models.Model):
-    _inherit = 'product.product'
+    _inherit = 'product.template'
```

Comment:
Yes, follow this migration guide: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-9.0